1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:09.560366 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:20:13.846385       1 horizontal.go:227] failed to query scale subresource for Deployment/default/dj-app-deployment: deployments/scale.apps \"dj-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:20:13.846641       1 event.go:291] \"Event occurred\" object=\"default/dj-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"dj-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:20:13.855033       1 horizontal.go:227] failed to query scale subresource for Deployment/default/fl-app-deployment: deployments/scale.apps \"fl-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:20:13.855398       1 event.go:291] \"Event occurred\" object=\"default/fl-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"fl-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:20:15Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:20:17.928224       1 client.go:360] parsed scheme: \"passthrough\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:17.932972 I | embed: rejected connection from \"127.0.0.1:62162\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:20:17.929459       1 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{https://127.0.0.1:2379  <nil> 0 <nil>}] <nil> <nil>}\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:20:17.929605       1 clientconn.go:948] ClientConn switching balancer to \"pick_first\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:17.932901       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:18.938307 I | embed: rejected connection from \"127.0.0.1:62174\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:18.939215       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:19.560633 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:20.293463 I | embed: rejected connection from \"127.0.0.1:62188\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:20.293925       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:23.180812 I | embed: rejected connection from \"127.0.0.1:62208\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:23.183393       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:28.060915 I | embed: rejected connection from \"127.0.0.1:62250\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:28.061477       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:29.560594 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:34.367856       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:34.367525 I | embed: rejected connection from \"127.0.0.1:62302\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:39.562102 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:42.892967       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:43.903665       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:20:45Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:47.262152       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:49.563482 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:20:54.343952       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:20:54.345922       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:59.208855 I | mvcc: store.index: compact 1438877\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:59.210607 I | mvcc: finished scheduled compaction at 1438877 (took 1.20742ms)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:20:59.561666 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:20:59.820317       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:21:01.338416       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
